### PR TITLE
include researchdelight hub in compass docs

### PR DESCRIPTION
**Pull request by jmunroe**
This is first step on adding references to the Research Delight hub to the team compass (see #644)

### CODE REVIEW

**Comment by jmunroe** [comment]
Other than adding this new page to index.md, is there any other configuration required add this new page?

Is 'References' the right place in the team compass for this content to live?

**Review by GeorgianaElena — COMMENTED** [review]
> Other than adding this new page to index.md, is there any other configuration required add this new page?

This should be enough. You can also check how the docs look like after each commit by clicking on the RTD action check at the end of the PR. For example, you can see the new page at https://2i2c-team-compass--745.org.readthedocs.build/reference/researchdelight/

> Is 'References' the right place in the team compass for this content to live?

I think I would have probably chosen the _"Managed JupyterHub Service"_  instead of the _"Reference"_ one. I believe although it's used by us and for demo, it's still a managed hub.
Also, I think of the _"Reference"_ section as a place that can be referenced and used to expand and explain part of the other sections.

**Comment by jmunroe** [comment]
Thanks @GeorgianaElena . Good call -- putting it within the "_Managed JupyterHub Service_" section does make more sense.

File: projects/managed-hubs/researchdelight.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The hub is hosted on AWS in us-west-2 and supports multiple profiles and machine types.
+
+## Images and environment
+
+See https://github.com/2i2c-org/researchdelight-image for additional information.
+
+## Data retention
+
+Research Delight is intended for day-to-day work but is not backed up and will generally track the bleeding edge of our infrastucture. It is important that all data, notebooks, and active work products are backed up in GitHub or another third-party service.
```
[the remark this answers is not in the document]
Comment (GeorgianaElena) [reply projects/managed-hubs/researchdelight.md:24]: Hope that's ok that I'm pushing the commit button myself. I want to test the auto assign workflow fix 😅

File: projects/managed-hubs/researchdelight.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## Images and environment
+
+See https://github.com/2i2c-org/researchdelight-image for additional information.
+
+## Data retention
+
+Research Delight is intended for day-to-day work but is not backed up and will generally track the bleeding edge of our infrastructure. It is important that all data, notebooks, and active work products are backed up in GitHub or another third-party service.
+
+
+
```
Comment (GeorgianaElena) [inline projects/managed-hubs/researchdelight.md:27]:
```suggestion
```

**Pull request merged**